index.html

```diff
@@ -33,5 +33,166 @@ <h1 class="top-name">ELDON RATZLAFF</h1>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+<!-- proyect 3 -->
+<div class="project3"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+<!-- proyect 4 -->
+<div class="project4"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+<!-- proyect 5 -->
+<div class="project5"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+
+
+
+<!-- proyect 6 -->
+<div class="project6"></div>
+<div class="proyect-foot1">
+    <p class="proyect-foot1-p">Project name goes here</p>
+    <ul class="proyect-foot1-ul">
+        <li class="proyect-foot1-li">HTML/CSS</li>
+        <li class="proyect-foot1-li">Ruby on Rails</li>
+        <li class="proyect-foot1-li">JavaScript</li>
+    </ul>
+    <div class="proyect-foot1-button">
+        <h3 class="proyect-foot1-text">See this project</h3>
+        <div class="proyect-foot1-img"></div>
+    </div>
+</div  >
+
+</section>
+
+
+<section class="about-section">
+    <div>
+        <h2 class="about-title">About
+        </h2>
+        <p class="about-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    </div>
+
+    <div class="get-mycv">
+        <h3 class="get-mycv-h">Get my CV</h3>
+        <div class="get-mycv-icon"></div>
+    </div>
```

Comment:
![image](https://user-images.githubusercontent.com/78251927/205417350-21d262de-c22c-4eda-8eb1-aae8c459e1c4.png)

- [ ] Please kindly make the button "Get my resume" a link `a` or `button` tag, that  way will be semantically correct.